lib/puppet/type/apache_directive.rb

```diff
@@ -52,4 +56,35 @@ def insync?(is)
   newparam(:target) do
     desc 'The config file to use'
   end
+
+  def self.title_patterns
+    identity = lambda { |x| x }
+    [
+      [
+        /^((\S+)\s+of\s+(.+))$/,
```

Comment:
Not that I'm right... but looking over in ap_postgresql it seems like maybe you could take a different approach here?

https://github.com/hercules-team/augeasproviders_postgresql/blob/master/lib/puppet/type/pg_hba.rb

Is namevar needed? or just that thigns that make it up.